features/comments_and_kudos/hidden_comments.feature

```diff
@@ -65,3 +65,26 @@ Feature: Comment hiding
       And I view the work "Popular Fic" with comments
     Then I should see "A suspicious comment"
       And I should not see "This comment has been hidden by an admin."
+
+  Scenario Outline: Embedded images in hidden comments are replaced with their URLs.
+    Given I am logged in as "author"
+    And I post the work "Popular Fic"
+    And I am logged out
+    And I am logged in as "commenter"
+    And I post the comment "OMG! <img src= 'https://example.com/image.jpg'>" on the work "Popular Fic"
+    And I am logged out
```

Comment:
Could you indent these one step, like the other And steps? Also, I think you can leave out the logging out step, logging in as super admin should also run that